pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -122,6 +126,9 @@ type APIClient struct {
 	defaultClientTimeout        time.Duration
 	defaultInformerTimeout      time.Duration
 	defaultInformerResyncPeriod time.Duration
+
+	// Config
```

Comment:
nit: i don't think this comment is useful

Reply:
We actually don't event need the `APIClient.config` attribute anymore (because of my last refacto). Getting rid of it ! Thanks !